quickstart/101-attestation-provider/main.tf

```diff
@@ -1,22 +1,10 @@
-terraform {
-
-  required_version = ">=0.12"
-    
-  required_providers {
-    azurerm = {
-      source = "hashicorp/azurerm"
-      version = "~>2.0"
-    }
-  }
-}
-
-provider "azurerm" {
-  features {}
+resource "random_pet" "rg-name" {
+  prefix    = var.resource_group_name_prefix
 }
 
 resource "azurerm_resource_group" "rg" {
-  name = var.resource_group_name
-  location = var.resource_group_location
+  name      = random_pet.rg-name.id
+  location  = var.resource_group_location
 }
 
 resource "azurerm_attestation_provider" "corpAttestation" {
```

Comment:
I know this line is irrelevant to this pr but maybe we should consider renaming this resource to `corp_attestation` to follow the [naming convention](https://github.com/terraform-linters/tflint/blob/master/docs/rules/terraform_naming_convention.md)?